workers/unity/Packages/com.improbable.gdk.core/Worker/DefaultWorkerConnector.cs

```diff
@@ -14,58 +16,151 @@ public abstract class DefaultWorkerConnector : WorkerConnector
         ///     Determines whether to connect via the locator.
         /// </summary>
         /// <returns>True, if should connect via the Locator, false otherwise.</returns>
-        protected override bool ShouldUseLocator()
+        protected override ConnectionService GetChosenService()
```

Comment:
Doc strings are incorrect here now.

Reply:
Also not a huge fan of `GetChosenService`. Maybe something like `GetConnectionService`?